drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -38,6 +38,25 @@ def self.find_definition_for_project(project, role:, version: nil)
     definitions.order(version: :desc).first
   end
 
+  # Validate JSON definition when loading, to ensure no duplicate link IDs
```

Comment:
Is this catching things like `form: [:fu, [:bar, :fu]]`
because the form would end up with two `input name="fu"`?

Reply:
Right. `link_id`s also need to be unique because they're used as references in `enable_when`. A link_id should be able to uniquely identify one question